src/components/InteractiveArticle.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+import Helmet from 'react-helmet';
+import NotFound from 'components/NotFound';
+
+export default class InteractiveArticle extends FalcorController {
+  static getFalcorPathSets(params) {
+    return [
+      // For now we only use the html part, the js and css parts are for further improvements
+      ['interactiveArticlesBySlug', params.articleSlug, ['html', 'js', 'css']],
+      // Fetch article data
+      [
+        'interactiveArticlesBySlug',
+        params.articleSlug,
+        ['title', 'teaser', 'html', 'published_at', 'issueNumber', 'category', 'slug', 'image'],
+      ],
+      ['interactiveArticlesBySlug', params.articleSlug, 'authors', { length: 10 }, ['name', 'slug']],
+
+    ];
+  }
+
+  render() {
+    if (this.state.ready) {
+      if (!this.state.data) {
+        return <NotFound />;
+      }
+      const articleSlug = this.props.params.articleSlug;
+      // Access data fetched via Falcor
+      const articleData = this.state.data.articlesBySlug[articleSlug];
+      // make sure article meta image has default
+      const articleMetaImage = articleData.image || 'https://thegazelle.s3.amazonaws.com/gazelle/2016/02/saadiyat-reflection.jpg';
+      const meta = [
+        // Search results
+        { name: 'description', content: this.props.teaser },
+
+        // Social media sharing
+        { property: 'og:title', content: `${articleData.title} | The Gazelle` },
+        { property: 'og:type', content: 'article' },
+        {
+          property: 'og:url',
+          content: `www.thegazelle.org/interactive/${articleData.slug}/`,
```

Comment:
@zanemountcastle I just updated this as it was previously the old URL format. This is the format we'll be using yeah? That's what I envisioned and changed it to in ArticlePreview.jsx anyway, so we need to keep it consistent no matter what.

Reply:
Yep, good catch